### PR TITLE
Fix authorization and json_body

### PR DESCRIPTION
**Pull request by songofroland**

### CODE REVIEW

File: tests/test_authz_Authorizer.py [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+import random
+import string
+
 import pytest
 
 from lbz.authz import Authorizer
 from lbz.exceptions import PermissionDenied, ServerError
 from lbz.misc import NestedDict
 
+ALL = "*"
```
Comment (pdyba) [inline tests/test_authz_Authorizer.py:12]: You could and should import that form auth

File: lbz/response.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!/usr/local/bin/python3.8
```
Comment (pdyba) [inline lbz/response.py:1]: why did You split communication into 2 files ?

File: tests/test_authz_Authorizer.py (hunk 1 above)
Comment (pdyba) [reply tests/test_authz_Authorizer.py:12]: that and below.

File: lbz/response.py (hunk 2 above)
Comment (songofroland) [reply lbz/response.py:1]: To avoid circular imports

**Review by pdyba — APPROVED** [review]

**Pull request merged**